terraform/mesh_lambdas/ssm_config.tf

```diff
@@ -0,0 +1,33 @@
+resource "aws_ssm_parameter" "ca_cert" {
+  name  = "/${local.name}/mesh/MESH_CA_CERT"
+  type  = "String"
```

Comment:
Secrets should be secure strings?

Reply:
Yep, will be once I have done the secrets pattern, these are just placeholder resources